Add mount and yada

xhh opened the pull request:
In the "Dependency injection" section, added "mount", removed "duct" as it belongs to the "Web Framework" section and uses "Component".

Added "yada" to "Web Framework" and "RESTful API" sections.

razum2um commented:
@xhh you've mentioned yada in place of duct I think https://github.com/razum2um/awesome-clojure/pull/43/files#diff-04c6e90faac2675aa89e2176d2eec7d8R60

xhh commented:
@razum2um yada was mainly a REST web library before but now is a general web library, including building RESTful services. So I added it to both sections.

For duct, it is a web library but not a lifecycle tool like Component and mount, that's why I removed it.

razum2um commented:
merged via https://github.com/razum2um/awesome-clojure/commit/44b0d30b36b479ddf693a0a2ed760b868345321e
